client/src/actions/loginModalAction.js

```diff
@@ -0,0 +1,8 @@
+import { LOGIN_MODAL } from '@cd/store/actionTypes';
+
+export const setIsOpen = (isOpen) => {
+	return {
+		type: LOGIN_MODAL.SET_IS_OPEN,
+		payload: { isOpen },
```

Comment:
For keeping small task handler and avoid adding more tasks than it should have, this can be simplified as:
```
return {
		type: LOGIN_MODAL.SET_IS_OPEN,
		payload:  isOpen,
}
```
Let me know if this is fine with you guys.